packages/frame/src/react/__stories__/skills.story.js

```diff
@@ -0,0 +1,170 @@
+import { storiesOf } from '@storybook/react'
+import React, { useCallback, useEffect, useState } from 'react'
+
+import {
+  HomeIcon,
+  BrowseIcon,
+  PlaceholderIcon
+} from '@pluralsight/ps-design-system-icon'
+
+import Frame from '../index.js'
+import {
+  MockContent,
+  SideNav as MockSideNav,
+  TopNav as MockTopNav
+} from './shared.js'
+
+import { breakpoints } from '../../vars/index.js'
+
+import useMatchMedia from '../use-match-media.js'
+
+storiesOf('Frame|Examples/Skills', module).add('controlled', () => {
+  const largeMedia = useMatchMedia(`(min-width: ${breakpoints.large})`)
+
+  const [open, setOpen] = useState(() => largeMedia)
+  const [hovered, setHovered] = useState(false)
+
+  const toggle = () => setOpen(!open)
+  const handleMouseEnter = () => setHovered(true)
+  const handleMouseLeave = () => setHovered(false)
+
+  const hideNavItems = !open && !hovered
+
+  return (
+    <Frame
+      sidenav={
+        <SkillsSideNav
+          collapsed={hideNavItems}
+          hideLabels={hideNavItems}
+          onMouseEnter={handleMouseEnter}
```

Comment:
Looks like we're leaving it to implementers to handle hover auto-expand.  Just checking/observing.  Is that right?

This case looks kinda weird, but maybe it's not real world?:

![hoverEffectButNoVisibility](https://user-images.githubusercontent.com/552044/89328531-c50b5680-d64a-11ea-86a5-f391f9321eec.gif)

Reply:
Related, are we not building a decision to expand the sidenav on certain wide screen widths?:

<img width="2116" alt="superWideStillCollapsed" src="https://user-images.githubusercontent.com/552044/89328692-fedc5d00-d64a-11ea-9a1c-2a00d1cf3288.png">

Reply:
Here's another one, at risk of it being lost amongst the previous 2:

This screen seems pretty thin, yet the sidebar is not shown as an overlay.  Instead, it's pushing the content to the right?  Just want to double-check that this is desired:

<img width="939" alt="thinScreenButNoOverlay" src="https://user-images.githubusercontent.com/552044/89328876-42cf6200-d64b-11ea-8881-f20f94d8bd6a.png">

Reply:
it's hard to draw the line....but i've left the decision of open/closed up to the implementors. i can easy add options for controlled/uncontrolled if that would make sense

cc @sweeting do you want to double check the breakpoints with me